packages/web/server/index.ts

```diff
@@ -121,6 +123,52 @@ function wwwRedirect(req: express.Request, res: express.Response, nextAction: ()
     res.redirect('https://forum.celo.org/t/the-great-celo-stake-off-the-details/136')
   })
 
+  // Redirects for OpenPGP WKD https://gnupg.org/blog/20161027-hosting-a-web-key-directory.html
```

Comment:
Maybe change the comment as these are not redirects?

Reply:
Good catch!